tests/end-to-end/test_api_alloc.py

```diff
@@ -51,16 +51,16 @@ def setUp(self):
         count = 0
         while True:
             try:
-                result = self._df_api.loginByPassword(username, password)
+                self._df_api.loginByPassword(username, password)
                 break
             except BaseException:
                 pass
             count += 1
             # Try three times to authenticate
             assert count < 3
 
-        print("Attempt to login result")
-        print(result)
+        print("\nAttempted to login\n")
```

Comment:
This is not the behavior we want, it would be beneficial to still print the results. I'm not sure why the analyzer was complaining about this.

Reply:
Nevermind, the assertion should fail if there is a problem. In hind sight this seems fine.